doc/user/content/sql/create-materialized-view.md

```diff
@@ -22,6 +22,7 @@ query in memory. For more information, see [API Components: Materialized views](
 
 Field | Use
 ------|-----
+**TEMP** / **TEMPORARY** | Mark the materialized view as temporary.
```

Comment:
Can we include a link to the on-page anchor explaining this feature?

Reply:
yes! adding to both